environments:
  - name: env1
    nodes:
      - name: lidar_publisher
        qty: 2
        root_node: true
        terminal_node: false
        publishers:
          - name: lidar_raw_data
            qty: 2
            msg_size: 2M
            frequency: 5.0
      - name: lidar_transformer
        qty: 2
        root_node: false
        terminal_node: false
        publishers:
          - name: lidar_transfer_data
            qty: 2
            bandwidth: 2M
            frequency: 20.0
        subscribers:
          - name: lidar_raw_data_1
            node: lidar_publisher
          - name: lidar_raw_data_2
            node: lidar_publisher
      - name: lidar_reciever
        qty: 2
        root_node: false
        terminal_node: true
        subscribers:
          - name: lidar_transfer_data_1
            node: lidar_transformer
          - name: lidar_transfer_data_2
            node: lidar_transformer
